fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/generator/DefaultNullInjectGenerator.java

```diff
@@ -39,14 +39,9 @@ public final class DefaultNullInjectGenerator implements NullInjectGenerator {
 	public static final List<String> DEFAULT_NULLABLE_ANNOTATION_TYPES = Collections.unmodifiableList(
 		Arrays.asList(
 			"javax.annotation.Nullable",
+			"jakarta.annotation.Nullable",
```

Comment:
Please let me know if it's not appropriate to include jakarta annotations in this list.